dist/common/scripts/scylla_swap_setup

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 ScyllaDB
+#
+
+#
+# This file is part of Scylla.
+#
+# Scylla is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Scylla is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import sys
+import argparse
+from pathlib import Path
+from scylla_util import *
+
+if __name__ == '__main__':
+    if os.getuid() > 0:
+        print('Requires root permission.')
+        sys.exit(1)
+    parser = argparse.ArgumentParser(description='Configure swap for Scylla.')
+    parser.add_argument('--swap-directory',
+                        help='specify swapfile directory', required=True)
+    args = parser.parse_args()
+
+    if swap_exists():
+        print('swap already configured, exiting setup')
+        sys.exit(1)
+
+    memtotal = get_memtotal_gb()
+    swapsize = 16 if 16 < memtotal / 3 else memtotal / 3
```

Comment:
This can be a huge amount. Let's clamp it between 16 and 50.

Reply:
Better also convert it to an integer.

Reply:
The logic is
```
if 16g <  (memtotal / 3)
    return 16g
else
    return (memtotal / 3)
```
It shouldn't be bigger than 16GB.
referenced: https://docs.scylladb.com/faq/#do-i-need-to-configure-swap-on-a-scylla-node

But agree with it need to to convert an integer, will fix.